src/js/api/core-shim.js

```diff
@@ -119,7 +120,11 @@ Object.assign(CoreShim.prototype, {
             storage.track(coreModel);
 
             // Set the active playlist item after plugins are loaded and the view is setup
-            return this.updatePlaylist(coreModel.get('playlist'), coreModel.get('feedData'));
+            return this.updatePlaylist(coreModel.get('playlist'), coreModel.get('feedData'))
+                .catch((error) => {
+                    error.code += (error.code || 0) + SETUP_ERROR_LOADING_PLAYLIST;
```

Comment:
wouldn't this be adding `error.code` twice ?
`error.code += SETUP_ERROR_LOADING_PLAYLIST;`